### ISSUE TITLE
Qureka

### ISSUE THREAD

**Issue by hackthon2018** [SubhamTyagi/loco-answers#375]
Is Loko hack working on qureka latest version or not?
How to take the screenshots in the Qureka,swoo?
Is there any apps?

**Comment by dee9205** [comment]
Yes , there is virtual exposed app .we need to use clan app in it of all trivia game which has screenshot problem but. In virtual exposed app qureka does not allow to login

**Comment by hackthon2018** [comment]
@dee9205 then what is the way for the screenshots??
Which version of Qureka u r using ??

**Comment by hackthon2018** [comment]
Ok bro , it means that you are not playing Qureka

**Comment by dee9205** [comment]
Yeah brother, its so sad .

**Comment by hackthon2018** [comment]
Y can't u Try by other mail id

**Comment by dee9205** [comment]
It is not giving permission to log in ..in virtual exposed.

**Comment by royalchauhan** [comment]
I can give you the screenshot but the problem is that loko hack is not working in qureka.  Even with I
OCR

**Comment by hackthon2018** [comment]
U mean that Qureka new version doesn't work with OCR ??

**Comment by royalchauhan** [comment]
Yes it does not work with loko hack OCR.

**Comment by hackthon2018** [comment]
Then,how will u keep answers
Is there any app bro??

**Comment by royalchauhan** [comment]
Trying to find a new app but the level of qureka's questions has gone way
down and they are too easy now.

**Comment by dee9205** [comment]
What you want to say i can't understand

**Comment by royalchauhan** [comment]
I am saying that I am finding a new app and also the question quality of
qureka has gone very low which is good for us.

**Comment by dee9205** [comment]
Which app bro ...tell us too

**Comment by royalchauhan** [comment]
Bro I am searching for it and when I find it I will let you know

**Comment by dee9205** [comment]
Okay bro

**Comment by vidhanwani** [comment]
K bro
